helm/g8s-oauth2-proxy-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: oauth2-proxy
+  name: oauth2-proxy
+  namespace: kube-system
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      k8s-app: oauth2-proxy
+  template:
+    metadata:
+      labels:
+        k8s-app: oauth2-proxy
+    spec:
+      containers:
+      - image: quay.io/giantswarm/oauth2_proxy:2.2
```

Comment:
is this just retagging of mainstream image or are we doing some changes there?

Reply:
It's just a retagged image of [this one](https://hub.docker.com/r/a5huynh/oauth2_proxy/). There is no official image, and this is the most commonly used, unfortunalty they only provide the `latest` tag for the version we need, so it's not in the retagger yet.

Reply:
make sure to write this somewhere so we don't have to wonder where did it come from. maybe at least comment above the image line

Reply:
Sure, will do. Thinking about just having a own repo with a Dockerfile, since theres no official one anyways.

Reply:
you can put it in this repo, no need for a new one

Reply:
But then architect would push this as `g8s-oauth-proxy` right?